.github/workflows/build.yml

```diff
@@ -14,27 +14,34 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v2
 
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+
       - name: Prepare environment
         run: |
           rm -rf ./build/*
           mkdir ./build
 
+      - name: Install dependencies
+        run: yarn install
+
       - name: Check packaging
-        uses: docker://ghcr.io/oddstr13/docker-tizen-webos-sdk:webos-only
-        with:
-          args: >-
-            ares-package
-            --check
-            org.jellyfin.webos
+        run: yarn run check
```

Comment:
You are correct, that can easily be replaced with an invocation of `npm`

Reply:
Or we export PATH with nodes_modules ;)

Reply:
Ah, I was expecting the ares commands to be on the path, yea.

And I was referring to all the yarn commands which obviously (when looking at this diff, but not when unfamiliar with how yarn works) corresponds to the scripts section in package.json.

I would not have known to look at the package.json to figure out what the CI does, had it not been for the fact that it was all on the same page when reviewing here.

A comment in the CI file about where the actual commands can be found would go a long way, and a wrapper script similar to the current dev one to not have to deal with installing (the appropriate versions of) npm and yarn would go a long way.
(I know from experience that attempting to install node stuff via the system package manager yields a few hours of pain, and not getting closer to change that one line of code you where looking to test out, cause of toolchain version missmatches).

The best thing to do here, would be to make a new docker image based on node ones, and install the ares tools in it, so that the commands would be easily available with a docker wrapper script, for those of us with a working docker install, but not all the javascript packaging shenanigans :P


Documentation in in this PR, and new docker stuff for the future (probably with an issue todo, pointing back at the context in this PR)

Reply:
Updated the CI workflow with direct invocations.
I agree, a new docker image would help for local development, for people who don't want to have npm installed locally but already have docker.

For CI I consider it overkill tbh, cause it's just a single command at the start of the workflow, then it's just invoking ares-tools as usual (aka. exactly how it was before)

Reply:
That makes the CI look much better, thanks.